package.json

```diff
@@ -17,6 +17,7 @@
     "dotenv": "^10.0.0"
   },
   "devDependencies": {
+    "@sapphire/ts-config": "^3.0.0",
```

Comment:
Do we not need typescript to even start compiling typescript..?

Reply:
maybe it was removed while i was resolving merge conflicts